### PR TITLE
Example of working Visitor Prioritization Policy Activation code

### PR DESCRIPTION
**Pull request by Beezer**
The documentation was confusing and in places, wrong. I hope this code helps others to get this working smoothly.

### CODE REVIEW

**Comment by dariusk** [comment]
Thanks so much for the PR! I'm taking a look at the code now.

File: contrib/python/activate-vp.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+from akamai.edgegrid import EdgeGridAuth
+from config import EdgeGridConfig
+from urlparse import urljoin
+import urllib
+import pprint
+
+session = requests.Session()
+debug = False
+
+VPNAME = 'icass_vptest2'
```
Comment (dariusk) [inline contrib/python/activate-vp.py]: Could you change these to more generic names? Something like

```python
VPNAME = "YOUR_NAME_HERE"
VERSION = "1"
NETWORK = "YOUR_NETWORK_HERE"
```

**Comment by dariusk** [comment]
I'm seeing mixed single- and double-quotes too. If you could normalize them (doesn't really matter to me which way) that would be great too.

**Comment by Beezer** [comment]
Quotes are fixed. Name of the policy is now generic & comments added to make it clear what the other values should be. The "POST" code is now extracted into a convenience function for tidyness. "RuntimeException" changed to "RuntimeError". Can you tell I'm more used to Java than Python? :)

**Comment by dariusk** [comment]
Merged this into contrib!

I'm curious, can you tell me what was wrong about the Visitor Prioritization docs? I'm assuming you mean [these](https://developer.akamai.com/api/luna/config-visitor-prioritization-data/overview.html)? I can work with that team to get the docs fixed.

**Comment by Beezer** [comment]
Darius, I think the docs are already being fixed. I sent you an email to your Akamai email address explaining.

**Pull request merged**